save-backend/build.gradle.kts

```diff
@@ -4,6 +4,22 @@ import  org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 plugins {
     kotlin("jvm")
+    id("org.liquibase.gradle") version Versions.liquibaseGradlePlugin
```

Comment:
Shouldn't it be in root `build.gradle.kts`?

Reply:
Fixed